articles/concepts/Glossary.md

```diff
@@ -12,31 +12,42 @@ uid: microsoft.quantum.glossary
+|<a id="bell-state"></a>Bell state|One of four specific maximally [entangled](xref:microsoft.quantum.glossary#entanglement) [quantum states](xref:microsoft.quantum.glossary#quantum-state) of two qubits. The four states are defined $\ket{\beta_{ij}} = (\mathbb{I} \otimes X^iZ^j) (\ket{00} + \ket{11}) / \sqrt{2}$. A Bell state is also known as an [EPR pair](xref:microsoft.quantum.glossary#epr).|
+|<a id="bloch"></a>Bloch sphere|A graphical representation of a single-[qubit](xref:microsoft.quantum.glossary#qubit) [quantum state](xref:microsoft.quantum.glossary#quantum-state) as a three-dimensional real-valued vector. For more information, see  [Visualizing Qubits and Transformations using the Bloch Sphere](xref:microsoft.quantum.concepts.qubit#visualizing-qubits-and-transformations-using-the-bloch-sphere).|
+|<a id="callable"></a>Callable|An [operation](xref:microsoft.quantum.glossary#operation) or [function](xref:microsoft.quantum.glossary#function) in the Q# language. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="clifford"></a>Clifford group|The set of operations that occupy the octants of the [Bloch sphere](xref:microsoft.quantum.glossary#bloch) and effect permutations of the [Pauli operators](xref:microsoft.quantum.glossary#pauli). These include the operations [$X$](xref:microsoft.quantum.intrinsic.x), [$Y$](xref:microsoft.quantum.intrinsic.y), [$Z$](xref:microsoft.quantum.intrinsic.z), [$H$](xref:microsoft.quantum.intrinsic.h) and [$S$](xref:microsoft.quantum.intrinsic.s).|
+|<a id="controlled"></a>Controlled|A quantum [operation](xref:microsoft.quantum.glossary#operation) that takes one or more [qubits](xref:microsoft.quantum.glossary#qubit) as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|
+|<a id="dirac"></a>Dirac Notation|A symbolic shorthand that simplifies the representation of [quantum states](xref:microsoft.quantum.glossary#quantum-state), also called *bra-ket* notation.  The *bra* portion represents a row vector, for example  $\bra{A} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix}$ and the *ket* portion represents a column vector, $\ket{B} = \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix}$.  Bras and kets can be combined, for example $\braket{A \| B} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix} \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix} = A{_1}B{_1} + A{_2}B{_2}$, or can represent the same quantum state in different ways, for example, $\ket{+} = H \ket{0} = \frac{1}{\sqrt{2}} \left(\ket{0} + \ket{1} \right) = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|
```

Comment:
I'm having a bit of trouble with the phrasing in the explanation of Dirac Notation, especially that "can represent the same quantum state in different ways" is not a scientific description. The example given is more to show that a quantum state can be decomposed into other quantum states, rather than explaining Diract notation. It may be better just to remove it or say it can be used to represent any state in a quantum system. Also the example of them being combined is only the inner product. There's also the outer product and generally speaking they act as operators. So perhaps say they can represent individual states or be combined to act as operators. (And there is an "operator application" in the glossary.)